bindings/wasm/src/tangle/client.rs

```diff
@@ -240,4 +254,7 @@ impl From<Client> for WasmClient {
 extern "C" {
   #[wasm_bindgen(typescript_type = "Promise<Client>")]
   pub type PromiseClient;
+
+  #[wasm_bindgen(typescript_type = "Promise<boolean>")]
+  pub type PromiseBoolean;
```

Comment:
There is already a `PromiseBool` in the Wasm bindings, re-use that to avoid duplicate definitions.

Reply:
That one is located in `account -> storage -> traits` it didn't feel right to reuse it. It should be moved to `common -> traits` but that is still not merged from #815. I didn't create a file with the same name to avoid merge conflicts.

Reply:
It should be moved to common, we can deal with the merge conflicts later.